content/_data/changelogs/lts.yml

```diff
@@ -1791,7 +1791,7 @@
     pull: 3542
     issue: 52534
     message: >
-      Allow <code>java.time.Ser</code> for use in XStream (XML serialization) and Remoting (agent communication).
+      Allow <code>java.time.Set</code> for use in XStream (XML serialization) and Remoting (agent communication).
```

Comment:
Wrong: https://github.com/jenkinsci/jenkins/blob/1a42044347502c59cd9c39d1be67f654df02908a/core/src/main/resources/jenkins/security/whitelisted-classes.txt#L57

Reply:
Ooops, I caught most of these.